make getting started page neater to read (#1657)
AztecProtocol/aztec-packages: docs/docs/dev_docs/getting_started/sandbox.md

## I have the Sandbox running, show me how to use it!

> We will deploy and use a private token contract on our Sandbox. Writing the contract itself is out of scope for this tutorial, we will use the Private Token Contract supplied as one of the example contracts.

Reviewer: ```suggestion
We will deploy a private token contract, and send tokens privately, using the Sandbox.

> Writing the contract itself is out of scope for this tutorial, we will use a Private Token Contract which as been pre-supplied as an example. See [here](LINK) for more information on how to write contracts for Aztec.
```

The following should work for MacOS, Linux or even WSL2 Ubuntu under Windows. 

Let's create an empty project called `private-token`. If you are familiar with setting up Typescript projects then you can skip to step 6.

Although both `yarn` and `npm` would work, this example uses `yarn`. Open the terminal and do the following

1. Ensure node version is 18 or more by running
```sh
node -v 
```

2. Initialize a yarn project
```sh

> ~$ mkdir private-token

Reviewer: For consistency, we should either always include the `~/path$` prefix, or never. (See the inconsistency between the command in 2. and the command in 3., for example). 
It is nice that we could show which directory the command is being executed from. But at the same time, it's not a copyable command, if it contains the prefix.

~$ cd private-token

> ~/private-token$ yarn init

Reviewer: See here if you wanted to do tabs for commands for each of npm or yarn. :) https://docusaurus.io/docs/markdown-features/code-blocks#multi-language-support-code-blocks

## I have the Sandbox running, show me how to use it!

We will deploy a private token contract, and send tokens privately, using the Sandbox.

> > Writing the contract itself is out of scope for this tutorial, we will use a Private Token Contract which as been pre-supplied as an example. See [here](LINK) for more information on how to write contracts for Aztec.

Reviewer: Whoops, sorry, my code suggestion was lazy and didn't actually provide you with a valid link. This'll need to be a valid link for the docs to actually build, via `yarn build`.

question private:
success Saved package.json
Done in 23.60s.
```

> 3. Create a `src` folder at the root of the repo:

Reviewer: ```suggestion
3. Create a `src` folder inside your new `private-token` directory:
```
